cypress/page/shoping-cart.page.ts

```diff
@@ -2,7 +2,7 @@ class ShopingCartPage {
   private proceedCheckoutBtn: string;
 
   constructor() {
-    this.proceedCheckoutBtn = '.cart_navigation span';
+    this.proceedCheckoutBtn = '.cart_navigation .standard-checkout';
```

Comment:
here you can use an attribute of a tag to indicate we are looking for the Proceed to checkout button